exts/cesium.omniverse/cesium/omniverse/api/globe_anchor.py

```diff
@@ -0,0 +1,14 @@
+from pxr.Sdf import Path
+from typing import Optional
+
+from ..utils.cesium_interface import CesiumInterfaceManager
+
+
+def anchor_xform_at_path(
+    path: Path, latitude: Optional[float] = None, longitude: Optional[float] = None, height: Optional[float] = None
+):
+    with CesiumInterfaceManager() as interface:
```

Comment:
Do we need to go through the old codebase and make sure we're grabbing the singleton elsewhere as a different quick PR?